ApplicationLibCode/FileInterface/RifEclipseInputFileTools.cpp

```diff
@@ -1054,6 +1054,111 @@ QString RifEclipseInputFileTools::faultFaceText( cvf::StructGridInterface::FaceT
     return "";
 }
 
+//--------------------------------------------------------------------------------------------------
+///
+//--------------------------------------------------------------------------------------------------
+void RifEclipseInputFileTools::parsePflotranInputFile( const QString& fileName, cvf::Collection<RigFault>* faults )
+{
+    QStringList gridSectionFilenames;
+
+    RiaLogging::info( "Looking for 'Pflotran' fault definitions in " + fileName );
+
+    {
+        // Find all referenced grdecl files in a pflotran input file, in the GRID section, example
+        /*
+            GRID
+                TYPE grdecl ../include/ccs_3df_kh.grdecl
+                TYPE grdecl ../include/ccs_3df_other.grdecl
+            END
+        */
+
+        QFile file( fileName );
+        if ( !file.open( QFile::ReadOnly ) ) return;
+
+        QString line;
+        bool    continueParsing  = true;
+        bool    foundGridKeyword = false;
+        bool    foundEndKeyword  = false;
+        do
+        {
+            line = file.readLine();
```

Comment:
Reduce `line` variable life time? Can be declared inside the do-while, I think.

Reply:
fixed